python/pyxbos/pyxbos/drivers/hvac/point_to_class.json

```diff
@@ -0,0 +1,1318 @@
+{
```

Comment:
Can you move these JSON files to a subdirectory for the building and determine the subdirectory to use from the driver configuration? That way we can more easily switch between buildings